### PR TITLE
Fixed type tg_id field in Client class

### PR DESCRIPTION
**Pull request by anmv**
Fixed type tg_id field in Client class #21

### CODE REVIEW

File: py3xui/client/client.py [hunk 1]
```diff
@@ -65,5 +67,5 @@ class Client(BaseModel):
     flow: str = ""
     limit_ip: int = Field(default=0, alias=ClientFields.LIMIT_IP)  # type: ignore
     sub_id: str = Field(default="", alias=ClientFields.SUB_ID)  # type: ignore
-    tg_id: str = Field(default="", alias=ClientFields.TG_ID)  # type: ignore
+    tg_id: Optional[int] = Field(default="", alias=ClientFields.TG_ID)  # type: ignore
```
Comment (iwatkot) [inline py3xui/client/client.py:70]: Default value is incompatible with the typing.
String can not be assigned as default value for Optional[int].

**Review by iwatkot — CHANGES_REQUESTED** [review]

File: py3xui/client/client.py (hunk 1 above)
Comment (anmv) [reply py3xui/client/client.py:70]: @iwatkot wow, really) default=None is ok ?

Comment (anmv) [reply py3xui/client/client.py:70]: panel 3x-ui return None if empty value in tg_id

Comment (anmv) [reply py3xui/client/client.py]: **1**
`tg_id: Optional[str] = Field(default=None, alias=ClientFields.TG_ID)`

_validation error for UiInbound
settings.clients.4.tgId
  Input should be a valid string [type=string_type, input_value=511122566, input_type=int]
    For further information visit https://errors.pydantic.dev/2.9/v/string_type_


**2**
`tg_id: Optional[int] = Field(default=None, alias=ClientFields.TG_ID)`

_validation error for UiInbound
settings.clients.0.tgId
  Input should be a valid integer, unable to parse string as an integer [type=int_parsing, input_value='', input_type=str]
    For further information visit https://errors.pydantic.dev/2.9/v/int_parsing_

 default panel return str type

**3**
`tg_id: Optional[int | str] = Field(default=None, alias=ClientFields.TG_ID)`

_Worked perfect_

Comment (anmv) [reply py3xui/client/client.py]: or default = "", i think this worked too

Comment (anmv) [reply py3xui/client/client.py]: @iwatkot

Comment (Goosegit11) [reply py3xui/client/client.py]: File "/root/py-scripts/lib/python3.10/site-packages/py3xui/client/client.py",
line 69, in Client
    tg_id: Optional[int | str] = Field(default=None, alias=ClientFields.TG_ID)
NameError: name 'Optional' is not defined

Comment (Goosegit11) [reply py3xui/client/client.py]: image

File: py3xui/client/client.py [hunk 2]
```diff
@@ -65,5 +67,5 @@ class Client(BaseModel):
     flow: str = ""
     limit_ip: int = Field(default=0, alias=ClientFields.LIMIT_IP)  # type: ignore
     sub_id: str = Field(default="", alias=ClientFields.SUB_ID)  # type: ignore
-    tg_id: str = Field(default="", alias=ClientFields.TG_ID)  # type: ignore
+    tg_id: Optional[int | str] = Field(default=None, alias=ClientFields.TG_ID)  # type: ignore
```
Comment (iwatkot) [inline py3xui/client/client.py:70]: Change to modern typehint int | str | None

File: py3xui/client/client.py [hunk 3]
```diff
@@ -1,5 +1,7 @@
 """This module contains the Client class which represents a client in the XUI API."""
 
+from typing import Optional
```
Comment (iwatkot) [inline py3xui/client/client.py:3]: Remove import

**Review by iwatkot — CHANGES_REQUESTED** [review]

**Comment by iwatkot** [comment]
@anmv, hello again!

I'm back, so my question would be:
If we set the default value to None, will it crash on the API side?
If yes, let's just leave something like
`int | str | None` with default `value=""`

What are your thoughts about it?

Thank you
Sincerely

File: py3xui/client/client.py (hunk 1 above)
Comment (iwatkot) [reply py3xui/client/client.py]: @Goosegit11

> NameError: name 'Optional' is not defined

This is weird, I can clearly see the import of the Optional from typing.
But still, we'll change it to modern type hints.

File: py3xui/client/client.py (hunk 2 above)
Comment (anmv) [reply py3xui/client/client.py:70]: > int | str | None

Ok, but api dont return "None"

**Comment by anmv** [comment]
@iwatkot i don't know about None as default value. I don't have error, when default value was error

**Comment by anmv** [comment]
> `int | str | None` with default `value=""`

ok, done

File: py3xui/client/client.py [hunk 4]
```diff
@@ -47,7 +47,7 @@ class Client(BaseModel):
         tg_id (str): The Telegram ID of the client. Optional.
         total_gb (int): The total amount of data transferred by the client in GB. Optional.
     """
-
+""""""
```
Comment (iwatkot) [inline py3xui/client/client.py:50]: Remove it, please

**Review by iwatkot — CHANGES_REQUESTED** [review]

**Comment by iwatkot** [comment]
@anmv remove the extra quotes and I'm merging it.

File: py3xui/client/client.py (hunk 4 above)
Comment (anmv) [reply py3xui/client/client.py:50]: @iwatkot wow, interesting :-D

**Pull request merged**